### PR TITLE
Update to strimzi 0.28 and kafka 3.1

### PR DESCRIPTION
**Pull request by BbolroC**
Signed-off-by: Hyounggyu Choi <Hyounggyu.Choi@ibm.com>
Fixes #

## Proposed Changes

- as per title
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e-common.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ readonly EVENTING_KAFKA_SECRET_TEMPLATE="300-kafka-secret.yaml"
 readonly EVENTING_KAFKA_CONFIG_TEMPLATE="300-eventing-kafka-configmap.yaml"
 
 # Strimzi installation config template used for starting up Kafka clusters.
-readonly STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.27.0.yaml"
+readonly STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.28.0.yaml"
```
Comment (matzew) [inline test/e2e-common.sh:76]: the PR says 3.1

this just bumps the operator itself

Comment (BbolroC) [reply test/e2e-common.sh:76]: Oh, forgot updating it. Thanks for catching it.

**Review by BbolroC — COMMENTED** [review]
Update and rename the file 100-kafka-ephemeral-triple-3.0.0.yaml to 100-kafka-ephemeral-triple-3.1.0.yaml

**Review by matzew — COMMENTED** [review]
/ok-to-test

**Comment by BbolroC** [comment]
hmm.. I will have a look into the test failures today. thanks!

**Comment by BbolroC** [comment]
@matzew I think this PR is ready to be merged. Thank you! ^^

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**